__tests__/nametag.test.tsx

```diff
@@ -1,16 +1,33 @@
 import '@testing-library/jest-dom'
-import { render, screen } from '@testing-library/react'
+import { render, screen, fireEvent } from '@testing-library/react'
+import { userEvent } from '@testing-library/user-event'
+
 import Page from '../app/nametag/page'
 
 jest.mock('next/navigation', () => jest.requireActual('next-router-mock'))
 jest.mock('../lib/zoomapi', () => jest.requireActual('../lib/fakezoomapi'));
 
 describe('Page', () => {
-  it('renders a heading', () => {
+  it('renders the heading and input fields', () => {
     render(<Page />)
 
-    const heading = screen.getByRole('heading', { level: 1 })
-
-    expect(heading).toBeInTheDocument()
+    expect(screen.getByRole('heading', { level: 1 })).toBeInTheDocument()
```

Comment:
Heading is less important than the other ones; I think it's OK to remove it from this test now that we have more meaningful test cases.